core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -225,9 +228,12 @@ public static String getAppVersion() {
         return appVersion;
     }
 
+    @SuppressWarnings({"UnusedReturnValue", "WeakerAccess"})
     public static final class Options {
 
-        private String configFileName = "mobile-services.json";
+        public static final String DEFAULT_CONFIG_FILE_NAME = "mobile-services.json";
+
+        private String configFileName = DEFAULT_CONFIG_FILE_NAME;
```

Comment:
Do we really need this redundant?

Reply:
I agree it is a bit redundant in this case. However I think it is a good practice to extract constants out of the code, always. In this case it is importante to make clear this string is a default value for the entire application, that is not going to change ever. Actually I would put it out of MobileCore even. It seems to me like a too relevant piece of string to be just hardcoded.

Reply:
Plus, this way it can be tested more robustly.

Reply:
But why not just use `DEFAULT_CONFIG_FILE_NAME` in the code instead of `configFileName`?